lib/LaTeXML/Package.pm

```diff
@@ -817,9 +838,9 @@ sub RefStepID {
 
 sub ResetCounter {
   my ($ctr) = @_;
-  AssignValue('\c@' . $ctr   => Number(0), 'global');
-  AssignValue('\c@UN' . $ctr => Number(0), 'global');    # ?????
-  DefMacroI(T_CS("\\\@$ctr\@ID"), undef, Tokens(Explode(LookupValue('\c@' . $ctr)->valueOf)),
+  AssignRegister('\c@' . $ctr   => Number(0), 'global');
+  AssignRegister('\c@UN' . $ctr => Number(0), 'global') unless $ctr =~ /^UN/;    # ?????
```

Comment:
I see there was an `# ?????` comment before, but now it has a new `unless $ctr=~/^UN/` added... Could I request a more verbose comment explaining the intention?

Reply:
ok, done